sharpy/linear/src/assembly.py

```diff
@@ -1155,26 +1156,43 @@ def dfunstdgamma_dot(Surfs):
     return DerList
 
 
-def wake_prop(Surfs, Surfs_star, use_sparse=False, sparse_format='lil'):
+def wake_prop(MS, use_sparse=False, sparse_format='lil', settings=None):
     """
     Assembly of wake propagation matrices, in sparse or dense matrices format
 
-    Note: wake propagation matrices are very sparse. Nonetheless, allocation
-    in dense format (from numpy.zeros) or sparse does not have important
-    differences in terms of cpu time and memory used as numpy.zeros does
-    not allocate memory until this is accessed.
+    Note:
+        Wake propagation matrices are very sparse. Nonetheless, allocation
+        in dense format (from numpy.zeros) or sparse does not have important
+        differences in terms of cpu time and memory used as numpy.zeros does
+        not allocate memory until this is accessed
+
+    Args:
+        MS (MultiSurface): MultiSurface instance
+        use_sparse (bool (optional)): Use sparse matrices
+        sparse_format (str (optional)): Use either ``csc`` or ``lil`` format
+        settings (dict (optional)): Dictionary with aerodynamic settings containing:
+            cfl1 (bool): Defines if the wake shape complies with CFL=1
+            dt (float): time step
+            vel_gen: velocity generator
```

Comment:
I see that the call to `settings['vel_gen']` is commented out, is it required? Also, speaking of the commented code below in lines 1230+, would you be happy if we got rid of it (provided all works)?

Reply:
You are right, I have just cleaned it up